ecosystem/lava-sdk/src/util/create-key.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  makeCosmoshubPath,
+  rawSecp256k1PubkeyToRawAddress,
+} from "@cosmjs/amino";
+import {
+  Bip39,
+  Random,
+  EnglishMnemonic,
+  Slip10,
+  Slip10Curve,
+  Secp256k1,
+} from "@cosmjs/crypto";
+import { toBech32 } from "@cosmjs/encoding";
+import elliptic from "elliptic";
+import axios from "axios";
+import { uuid } from "uuidv4";
+
+export const MAX_ATTEMPTS = 10;
+
+export enum DeveloperKeyStatus {
+  ERROR = "error",
+  PENDING = "pending",
+  SYNCED = "synced",
+  DELETING = "deleting",
+}
+
+export type GeneratedKeyType = {
+  address: string;
+  privateHex: string;
+  mnemonicChecked: EnglishMnemonic;
+};
+
+export type DeveloperKeyParams = {
+  projectKey: string;
+  name: string;
+};
+
+const GW_BACKEND_URL = "https://gateway-master.lava-cybertron.xyz/sdk";
```

Comment:
instead of constant lets pass it as an optional parameter to each function.
default value should be production url.